### PR TITLE
Attrcase

### PR DESCRIPTION
**Pull request by bung87**

### CODE REVIEW

File: karax/karax.nim [hunk 1]
```diff
@@ -219,7 +232,7 @@ proc same(n: VNode, e: Node; nesting = 0): bool =
   elif n.kind == VNodeKind.vthunk or n.kind == VNodeKind.dthunk:
     # we don't check these:
     result = true
-  elif toTag[n.kind] == e.nodename:
+  elif cmpIgnoreCase($toTag[n.kind], $e.nodename) == 0:
```
Comment (Araq) [inline karax/karax.nim:235]: Too slow, we need some JS specific code.

File: karax/karax.nim [hunk 2]
```diff
@@ -152,7 +152,10 @@ proc getVNodeById*(id: cstring; kxi: KaraxInstance = kxi): VNode =
   if kxi.byId.contains(id):
     result = kxi.byId[id]
 
-proc toDom*(n: VNode; useAttachedNode: bool; kxi: KaraxInstance = nil): Node =
+const KnownNS = {"svg":"http://www.w3.org/2000/svg","math":"http://www.w3.org/1998/Math/MathML"}.toTable()
```
Comment (Araq) [inline karax/karax.nim:155]: No... Too slow and too special cased.

File: karax/karax.nim [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ## Karax -- Single page applications for Nim.
 
-import kdom, vdom, jstrutils, compact, jdict, vstyles
-
+import kdom, vdom, jstrutils, compact, jdict, vstyles, tables
+import strutils
```
Comment (Araq) [inline karax/karax.nim:4]: Both of these imports are usually bad for Nim JS.

File: karax/karax.nim (hunk 1 above)
Comment (bung87) [reply karax/karax.nim:235]: ok ,get it.

**Comment by geotre** [comment]
Related #100, #170. @bung87 if setRenderer change happens in #207, do you want to adjust this PR so we can merge it?

**Comment by geotre** [comment]
I'm going to close this for now, but please let me know if you want to reopen it @bung87

**Pull request closed without merging**